Containerfile.test

```diff
@@ -16,7 +16,7 @@ RUN dnf update -qy && \
     && \
     dnf clean all
 COPY ./dist/apex /bin/apex
-ENTRYPOINT [ "/bin/apex" ]
+CMD [ "/bin/bash" ]
```

Comment:
Honestly, `/bin/sh` might be preferable in all cases, so we don't end up relying on bash-isms that make our tests not portable